.circleci/config.yml

```diff
@@ -1619,17 +1619,19 @@ workflows:
       - t_ext: *job_native_test_ext_gnosis
       - t_ext: *job_native_test_ext_zeppelin
       - t_ext: *job_native_test_ext_ens
-      - t_ext: *job_native_test_ext_trident
-      - t_ext: *job_native_test_ext_euler
+      # NOTE: The external tests below were commented because they
+      # depend on a specific version of hardhat which does not support shanghai EVM.
```

Comment:
This should either be above *all* of the commented test runs or you move the others directly here. The idea of the comment is to see for which ones we did this without disecting git comments :-).
Collecting all of them together here is easiest.